module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -549,11 +549,21 @@ public function getHolding($bibId, array $patron = null, array $options = [])
                 );
                 $locationId = $item->effectiveLocationId;
                 $locationName = $this->getLocationName($locationId);
+                // concatenate enumeration fields if present
+                $enum = join(
```

Comment:
`join` is an alias of `implode`, and VuFind uses `implode` throughout the code. Probably best to switch to that for consistency.